MAUI/Themes/Keys.md

```diff
@@ -107,3 +107,191 @@ This page lists the keys for each control and the element to which it is mapped
+            <br/>
+        </td>
+        <td>
+           SfAccordionTheme 
+            <br/>
+            <br/>
+        </td>
+        <td>    
+            By merging this key in application resources, you can customize the appearance of SfAccordion without merging common theme resource and control style resource dictionaries.
+			
+{% highlight xaml %}
+
+<Application xmlns:syncTheme="clr-namespace:Syncfusion.Maui.Themes;assembly=Syncfusion.Maui.Core"
+             ...>
+ <Application.Resources>
+    <ResourceDictionary>
+        <ResourceDictionary.MergedDictionaries>
+            <syncTheme:SyncfusionThemeResourceDictionary />
+            <ResourceDictionary>
+                <x:String x:Key="SfAccordionTheme">CommonTheme</x:String>
+                <Color x:Key="SfAccordionNormalHeaderBackground">Blue</Color>
+                <Color x:Key="SfAccordionNormalHeaderIconColor">LightGreen</Color>
+            </ResourceDictionary>
+        </ResourceDictionary.MergedDictionaries>
+    </ResourceDictionary>
+ </Application.Resources>
+
+....
+
+</Application>
+
+{% endhighlight %}
+            <br/>
+            <br/>
+        </td>
+    </tr>
+    <tr>
+        <td> SfAccordionNormalHeaderBackground <br/><br/></td>
+        <td> Background color of header in SfAccordion.<br/><br/></td>
+    </tr>
+    <tr>
+        <td> SfAccordionHoverHeaderBackground <br/><br/></td> 
+        <td> Background color of header when it is in hover state.<br/><br/></td>
+    </tr>
+    <tr>
+        <td> SfAccordionFocusedHeaderBackground <br/><br/></td> 
+        <td> Background color of header when it is in focused state.<br/><br/></td>
+    </tr>
+    <tr>
+        <td> SfAccordionHeaderRippleBackground <br/><br/></td> 
+        <td> Background color of header when it is in pressed state.<br/><br/></td>
+    </tr>
+    <tr>
+        <td> SfAccordionNormalHeaderIconColor <br/><br/></td>
+        <td> Color of the SfAccordion Header Icon.<br/><br/></td>
+    </tr>
+    <tr>
+        <td> SfAccordionHoverHeaderIconColor <br/><br/></td> 
+        <td> Color of header icon when it is in hover state.<br/><br/></td>
+    </tr>
+    <tr>
+        <td> SfAccordionPressedHeaderIconColor <br/><br/></td> 
+        <td> Color of header icon when it is in pressed state.<br/><br/></td>
+    </tr>
+    <tr>
+        <td> SfAccordionFocusedHeaderIconColor <br/><br/></td> 
+        <td> Color of header icon when it is in focused state.<br/><br/></td>
+    </tr>
+    <tr>
+        <td> SfAccordionExpandedItemStroke <br/><br/></td> 
+        <td> Stroke color of expanded accordion item.<br/><br/></td>
+    </tr>
+    <tr>
+        <td> SfAccordionExpandedItemStrokeThickness <br/><br/></td> 
+        <td> Stroke thickness of expanded accordion item.<br/><br/>
+        </td>
+    </tr>
+    <tr>
+        <td> SfAccordionFocusedItemStroke <br/><br/></td> 
+        <td> Stroke color of the header when it is in the focused state while using key navigation.<br/><br/></td>
```

Comment:
Stroke color of the header when it is in focused state during keyboard navigation.

Reply:
Modified